srcpkgs/cloudflared/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'cloudflared'
+pkgname=cloudflared
+version=2021.7.0
+revision=1
+build_style=go
+go_import_path=github.com/cloudflare/cloudflared
+go_package="${go_import_path}/cmd/cloudflared"
+go_ldflags="-X \"main.Version=${version}\" -X \"main.BuildTime=$(date -u '+%Y-%m-%d-%H:%M UTC')\""
+short_desc="Argo Tunnel client"
+maintainer="Elie ROUDNINSKI <xademax@gmail.com>"
+license="custom:cloudflare-servicesagreement"
```

Comment:
[this](https://github.com/cloudflare/cloudflared/blob/master/LICENSE#L37-L41) read pretty nonfree to me

Reply:
And?

Reply:
Done :+1: